services/graphite/config.go

```diff
@@ -38,6 +38,7 @@ type Config struct {
 	BindAddress      string        `toml:"bind-address"`
 	Database         string        `toml:"database"`
 	Enabled          bool          `toml:"enabled"`
+	WriteSkip        bool          `toml:"write-skip"`
```

Comment:
It seems odd to have this in the config though since the use case for this is mainly for developers.  Won't running `influxd config` print it out as the sample config?  Usually things like this are enabled using an env var in other systems.  e.g. `SKIP_WRITES=1 influxd`